cmd/kube-apiserver/app/server.go

```diff
@@ -227,6 +227,10 @@ func (s *APIServer) verifyClusterIPFlags() {
 	if s.ServiceClusterIPRange.IP == nil {
 		glog.Fatal("No --service-cluster-ip-range specified")
 	}
+	var ones, bits = s.ServiceClusterIPRange.Mask.Size()
+	if bits-ones > 20 {
```

Comment:
Why is this not simply `ones < 12`?  No confusion there :)

Also 12 should be a `const minServiceCIDRMask` or something